scripts/document.sh

```diff
@@ -50,4 +50,7 @@ REPLACE_REGEXP+='s/BRANDLESS_DOCSET_TITLE/Navigation SDK for iOS $1/, '
 REPLACE_REGEXP+="s/<span class=\"kt\">(${DIRECTIONS_SYMBOLS})<\/span>/<span class=\"kt\"><a href=\"${BASE_URL//\//\\/}\/directions\/${DIRECTIONS_VERSION}\/Classes\/\$1.html\">\$1<\/a><\/span>/, "
 
 find ${OUTPUT} -name *.html -exec \
-    perl -pi -e "$REPLACE_REGEXP" {} \;
\ No newline at end of file
+    perl -pi -e "$REPLACE_REGEXP" {} \;
+
+
+echo $SHORT_VERSION > ./latest_version
```

Comment:
This puts the file in the current working directory, which isn’t the same as the output directory (`$OUTPUT`).